config/kubernetes/org.yaml

```diff
@@ -877,6 +876,7 @@ members:
 - robinpercy
 - robscott
 - rojkov
+- rohitagarwal003
```

Comment:
oops, this needs to be sorted alphabetically :)

Reply:
Done.